proteuslib/unit_models/reverse_osmosis_1D.py

```diff
@@ -585,7 +587,10 @@ def initialize(blk,
             state_args=permeate_block_args)
 
         init_log.info_high("Initialization Step 1 Complete.")
-
+        if degrees_of_freedom(blk) != 0:
+            raise Exception(f"Initialization was called on {blk} "
+                            f"but it had {degrees_of_freedom(blk)} degrees of freedom "
```

Comment:
```suggestion
                            f"but it had {degrees_of_freedom(blk)} degree(s) of freedom "
```
Might as well get this one too